server/parser_test.go

```diff
@@ -15,6 +15,7 @@ package server
 
 import (
 	"bytes"
+	"errors"
```

Comment:
Bump copyright date top of file to "-2020"

Reply:
fixed